### DISCUSSION TITLE
Is it possible to tune different algorithms in one study?

### DISCUSSION THREAD

**Discussion by susmithakreddy** [optuna/optuna#6190] [Q&A]
Hello everyone,I recently stumbled upon Optuna while researching my usecase and by god, is it a cool piece of code!

Anyway, I have a usecase where i need to run multiple vision models and see which gives the best results.Now,is it possible to tune all these models in one study and compare the results and if thats not possible,run parallel studies for each algorithm and compare the results of these?

Sorry if the queations' a silly one.Im new to the domain😅

**Comment by nzw0301** [comment]
I think you can do either way. For single study case, if very different algorithms share the same parameter name (such as `learning_rate` in gradient boosting decision tree and DNN), having independent study per algorithm will be more appropriate.
